RELEASE-NOTES.md

```diff
@@ -21,6 +21,7 @@
 * Changing imports in function/procedure section in `snowflake.yml` will cause the definition update on replace
 * Adding `--pattern` flag to `stage list` command for filtering out results with regex.
 * Fixed snowpark build paths for builds with --project option (fixed empty zip issue).
+* `stage copy` from stage will reproduce the directory structure locally.
```

Comment:
Hm, this is potentially breaking change? Should we introduce this behind a flag `-r/--recursive` or something?

Reply:
Should we also add a warning for default use? In this way we will be able to prompt users to use `--recursive` always and later make it default behavior. What do you think?

Reply:
Good idea, but I wonder if it won't be a breaking change

Reply:
Warning is never a breaking change. We should utilize the cli_console.warning so it is automatically suppressed in formatted outputs

Reply:
With `--format JSON` no warning will appear, so maybe it will not be a problem? 🤔

Reply:
Added warning